mutiny-core/src/ldkstorage.rs

```diff
@@ -93,8 +99,10 @@ impl MutinyNodePersister {
     ) -> Result<Vec<(BlockHash, ChannelMonitor<InMemorySigner>)>, io::Error> {
         // Get all the channel monitor buffers that exist for this node
         let suffix = self.node_id.as_str();
-        let channel_monitor_list: HashMap<String, Vec<u8>> =
-            self.storage.scan(MONITORS_PREFIX_KEY, Some(suffix));
+        let channel_monitor_list: HashMap<String, Vec<u8>> = self
+            .storage
+            .scan(MONITORS_PREFIX_KEY, Some(suffix))
+            .map_err(|_| io::ErrorKind::Other)?;
```

Comment:
`io::ErrorKind` or `MutinyStorageError::Other`?

Reply:
this function returns `io::ErrorKind` so think should keep it that way at least for this PR, if we want to change the error types of these functions probably more suited for a follow up